lib/cloudregister/registerutils.py

```diff
@@ -337,6 +337,50 @@ def get_current_smt():
     return smt
 
 
+# ----------------------------------------------------------------------------
+def get_instance_data(config):
+    """Run the configured instance data collection command and return
+       the result or none."""
+    instance_data = None
+    if (
+            config.has_section('instance') and
+            config.has_option('instance', 'dataProvider')
+    ):
+        instance_data_cmd = config.get('instance', 'dataProvider')
+        cmd = instance_data_cmd.split()[0]
+        if cmd != 'none':
+            if cmd[0] != '/':
```

Comment:
Given the comparisons, cmd is assumed to be a string. Suggest using startswith which I think makes this more clear when reading.

```
if not cmd.startswith('/')
```